content/pandas/concepts/built-in-functions/terms/concat/concat.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: '.concat()'
+Description: 'Concatenates multiple Dataframes or Series along a particular axis.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Functions'
+  - 'Pandas'
+CatalogContent:
+  - 'paths/data-science'
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science-foundations'
```

Comment:
```suggestion
  - 'paths/data-science'
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science-foundations'
```
We prefer to keep 2 CatalogContent here:
- Free course
- Pro course/path
Can you modify it accordingly?